### DISCUSSION TITLE
Adding new features to waybar

### DISCUSSION THREAD

**Discussion by ghost** [Alexays/Waybar#1817] [Q&A]
I have a few specific features I want to try and implement into waybar, i.e. hiding waybar, "smart-icons", etc.. I have some questions about the code and I am wondering if there is any way I could talk with you directly about the source code to better understand the internals of waybar?

Shameless plug : we created a discord wayland server ( https://discord.gg/Ve45BqX ) if you use discord.

**Comment by Alexays** [comment]
Sorry for the delay, I'm quite out of time, but we can talk here, that way if people are asking the same questions, they'll get their answers.

**Comment by ghost** [comment]
No worries. I am a bit behind as well.

I was wanting to try and build in a autohide feature into the code.

I was able to add floating modules by changing the .css file which took a little while to figure out, but definitely was worth the time. I had more ideas but it's been a while. 
![screenshot](https://user-images.githubusercontent.com/44128193/79626718-c8bdb080-8121-11ea-9cb3-1f5dcdea6fb3.png)

Shameless plug: There is a Wayland Discord server where people are sharing their own waybar addons. May be worthwhile?
